disdrodb/L0/templates/FRANCE/temp_parser_ACTRIS-CCRES.py

```diff
@@ -387,6 +387,7 @@
 
```

Comment:
This file must be removed ... also the folder templates

Reply:
sorry, it is done yet